### PR TITLE
Profile schema

### PR DESCRIPTION
**Pull request by c-castillo**

### CODE REVIEW

File: profile.graphql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  email: String
+  location: String
+  website: String
+  github: String
+  birthday: String
+  work: Work
+  education: Education
+}
+
+type Mutation {
```
Comment (oed) [inline profile.graphql]: We won't really be able to support mutations at this point.

File: profile.graphql [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  description: String
+  spiritEmoji: Emoji
+  email: String
+  location: String
+  website: String
+  github: String
+  birthday: String
+  work: Work
+  education: Education
+}
```
Comment (oed) [inline profile.graphql]: Does this mean that we need to update the schema for every new entry? Is it possible to have generic params? If a dapp adds a new one it should show up automatically when you query the public profile.

Comment (c-castillo) [reply profile.graphql]: I implement this from the assumption the profile parameters are fixed (took them from the dapp)
I can take a look if schemas support metaparameters though.

File: profile.graphql (hunk 1 above)
Comment (c-castillo) [reply profile.graphql]: and how do we update profiles on the graphql endpoint?

Comment (oed) [reply profile.graphql]: I don't think we will support that at this point. It basically requires users to sign data in the orbitdb entry format which adds a bunch of complexity. Let's just focus on querying/caching for now :)

File: profile.graphql (hunk 2 above)
Comment (oed) [reply profile.graphql]: Thanks!

Comment (c-castillo) [reply profile.graphql]: It doesn't according to the documentation. So this could be a painpoint

Comment (oed) [reply profile.graphql]: ok, what happens if you query for a field that isn't present? You just get `null` back?

**Pull request closed without merging**